app/helpers/casino/sessions_helper.rb

```diff
@@ -51,9 +51,16 @@ def sign_out
     cookies.delete :tgt
   end
 
-  def log_failed_login(username)
+  def user_locked?(username)
+    CASino::User.where(username: username).to_a.any? do |user|
+      user.locked?
```

Comment:
Hmm... if there are the same usernames on several backends, all of them should be locked? So just fetching the first one would be okay for this check...?